src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.margin-form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.input {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  width: 100%;
+  height: 70px;
+  line-height: 22px;
+  padding-left: 62px;
+  box-sizing: border-box;
```

Comment:
move to `*` selector

Reply:
How can i move Height, Line-height, Padding-left to "* selector" if this components are unique to big and small searchbars?